CHANGELOG.md

```diff
@@ -1,14 +1,41 @@
 # Changelog
 
-## Unreleased
+## UNRELEASED
 
-###  Fixes
+### Adds
+
+* Relationships load more quickly.
```

Comment:
Please merge main into `keyboard-shortcut` also and let me know, and then I should be able to see just the relevant changes in this PR.

Reply:
(Right now the github pull request view shows many recent changes in main because your new branch is more up to date with `main` than the merge target is.)

Reply:
I'm sorry, I forgot to update the branch `feature/keyboard-shortcut` where this PR will be merged.